### DISCUSSION TITLE
bed plate size

### DISCUSSION THREAD

**Discussion by Mphasenauer** [mriscoc/Ender3V2S1#959] [General]
is there a way to get the bed size to 310x310?

**Comment by mriscoc** [comment]
Yes, check the wiki: 
https://github.com/mriscoc/Ender3V2S1/wiki/Physical-Settings-Menu
https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
![Physical settings](https://raw.githubusercontent.com/mriscoc/Ender3V2S1/Ender3V2S1-Released/screenshots/physical-settings-1.jpg)
